.github/workflows/ci.yml

```diff
@@ -1428,4 +1430,7 @@ jobs:
     steps:
       - name: Exit with error if some jobs are not successful
         run: exit 1
-        if: ${{ always() && (contains(needs.*.result, 'failure') || contains(needs.*.result, 'skipped') || contains(needs.*.result, 'cancelled')) }}
+        if: ${{ always() && github.repository_owner == 'scalar' && (contains(needs.*.result, 'failure') || contains(needs.*.result, 'cancelled')) }}
```

Comment:
Was the removal of the `skipped` condition when owner is `scalar` intentional?
The original behaviour was to exit with code 1 if any required job was skipped (e.g., due to user cancellation).